build.sh

```diff
@@ -38,11 +40,15 @@ install() {
   echo "Installing data..."
   mkdir -p $PKG_DATA_DIR
   mkdir -p $PKG_DOC_DIR
+  mkdir -p $PKG_LOG_DIR/$LOG_NAME
+  touch $PKG_LOG_DIR/$LOG_NAME/$LOG_NAME.log
 
   cp -pR defaults/ $PKG_DATA_DIR
   cp -pR library/ $PKG_DATA_DIR
   cp -pR meta/ $PKG_DATA_DIR
   cp -pR tasks/ $PKG_DATA_DIR
+  cp -pR examples/ $PKG_DATA_DIR
```

Comment:
We store examples in /usr/share/doc for other roles, jut FYI, if you want to have it also in /usr/share/ you can, up to you.

Reply:
I would want to use the ansible play there in the script, that is why I'm copying it.